app/services/import_actuals.rb

```diff
@@ -88,8 +89,12 @@ def create_actual
       attrs = @converter.to_h
       assign_default_values(attrs)
 
-      creator = CreateActual.new(activity: @activity, report: @report)
-      result = creator.call(attributes: attrs)
+      creator = if @converter.actual_class == Refund
+        CreateRefund.new(activity: @activity, report: @report, user: @uploader)
+      else
+        CreateActual.new(activity: @activity, report: @report)
```

Comment:
Just noticed that `CreateActual` can take a `user:` but is `nil` by default (hence the difference in params on L95 and L93). Is there a reason we've never set it on actuals before?

Reply:
No clue. I should probably set it?

Reply:
Possibly yes. We should ask the question more widely I think.